tests/virt_autotest/libvirt_host_bridge_virtual_network.pm

```diff
@@ -78,8 +85,17 @@ sub run_test {
     assert_script_run("virsh net-destroy vnet_host_bridge");
     save_screenshot;
 
+    #Restore guest before restore Network on sles11sp4 vm host
+    virt_autotest::virtual_network_utils::restore_guests() if (is_sle('=11-sp4'));
     #Restore Network setting
     virt_autotest::virtual_network_utils::restore_network($virt_host_bridge);
+
+    #After destroy created host bridge interface - br0 on sles11sp4 vm hosts
+    #Need tp reset up environment - br123 for virt_auto test
+    virt_autotest::virtual_network_utils::restore_standalone() if (is_sle('=11-sp4'));
+    #Need to reset guest system due to reset environment - br123 for
+    #virt_auto test on sles11sp4 vm host
+    virt_autotest::virtual_network_utils::reset_guests() if (is_sle('=11-sp4'));
 }
```

Comment:
Better to have others review the above changes. I am not clear about the details here.

Reply:
sure, got it. 
will update more clear details here

Reply:
Please check updated from restore_network func